climada/hazard/storm_europe.py

```diff
@@ -830,3 +831,106 @@ def _hist2prob(self, intensity1d, sel_cen, spatial_shift, ssi_args=None,
         ssi = self.calc_ssi(intensity=intensity_out, **ssi_args)
 
         return intensity_out[:, sel_cen], ssi
+
+
+def generate_WS_forecast_hazard(run_datetime = dt.datetime.today().replace(hour=0, 
```

Comment:
Please update the docstring to NumpyDoc format!

Reply:
Done